ui/app/AppLayouts/Profile/views/MyProfileView.qml

```diff
@@ -34,86 +35,221 @@ SettingsContentBase {
 
     property var communitiesModel
 
-    titleRowComponentLoader.sourceComponent: StatusButton {
-        objectName: "profileSettingsChangePasswordButton"
-        text: qsTr("Change Password")
-        onClicked: Global.openPopup(changePasswordModal)
-        enabled: !userProfile.isKeycardUser
+    titleRowComponentLoader.sourceComponent: RowLayout {
+        StatusButton {
+            objectName: "profileSettingsChangePasswordButton"
+            text: qsTr("Change Password")
+            onClicked: Global.openPopup(changePasswordModal)
+            enabled: !userProfile.isKeycardUser
+        }
+        StatusButton {
+            text: qsTr("Preview")
+            onClicked: Global.openPopup(profilePreview)
+        }
     }
 
-    dirty: settingsView.dirty
-    saveChangesButtonEnabled: settingsView.valid
+    dirty: (!descriptionPanel.isEnsName &&
+            descriptionPanel.displayName.text !== profileStore.displayName) ||
+            descriptionPanel.bio.text !== profileStore.bio ||
+            profileStore.socialLinksDirty ||
+            profileHeader.icon !== profileStore.profileLargeImage ||
+            priv.hasAnyProfileShowcaseChanges
+    saveChangesButtonEnabled: !!descriptionPanel.displayName.text && descriptionPanel.displayName.valid
 
-    onResetChangesClicked: {
-        settingsView.reset()
-        profilePreview.reload()
-    }
-    onSaveChangesClicked: {
-        settingsView.save()
-        profilePreview.reload()
-    }
+    onResetChangesClicked: priv.reset()
+
+    onSaveChangesClicked: priv.save()
 
     bottomHeaderComponents: StatusTabBar {
-        id: editPreviwTabBar
+        id: profileTabBar
+
         StatusTabButton {
+            width: implicitWidth
             leftPadding: 0
+            text: qsTr("Identity")
+        }
+
+        StatusTabButton {
             width: implicitWidth
-            text: qsTr("Edit")
+            text: qsTr("Communities")
         }
+
         StatusTabButton {
             width: implicitWidth
-            text: qsTr("Preview")
+            text: qsTr("Accounts")
+        }
+
+        StatusTabButton {
+            width: implicitWidth
+            text: qsTr("Collectibles")
+        }
+
+        StatusTabButton {
+            width: implicitWidth
+            text: qsTr("Assets")
+        }
+
+        StatusTabButton {
+            width: implicitWidth
+            text: qsTr("Web")
+        }
+    }
+
+    onVisibleChanged: if (visible) profileStore.requestProfileShowcasePreferences()
+    Component.onCompleted: profileStore.requestProfileShowcasePreferences()
+
+    readonly property var priv: QtObject {
```

Comment:
```suggestion
    readonly property var d: QtObject {
```
For consistency

Reply:
No, this will actually break the app :) SettingsContentsBase.qml already has such a property

Reply:
Yeah, the property name could affect base class. But it can be defined this way, still preserving our convention:
```
    readonly property var priv: QtObject {
        id: d
```
or
```
    readonly property var d_: QtObject {
        id: d
```